tests/app/questionnaire/test_router.py

```diff
@@ -457,7 +453,7 @@ def get_section_return_location_when_section_complete_section_summary_last_block
             self.metadata,
         )
 
-        routing_path = [Location(section_id="accommodation-section", block_id="proxy")]
+        routing_path = [RoutingPath(["proxy"], section_id="accommodation-section")]
```

Comment:
Where we're returning a full_routing_path we should update the variable names to be clear.